cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -283,3 +248,50 @@ def _serialize_entrance_exam(entrance_exam_module):
     return json.dumps({
         'locator': unicode(entrance_exam_module.location)
     })
+
+
+def add_entrance_exam_milestone(course_id, x_block):
```

Comment:
If this operation is intended for internal use only we should prefix the name with an underscore.  If it's not internal-only that's fine but we should add one or more unit tests to cover it, specifically.